tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
+    DIVISION_COLUMN_NAMES = {
```

Comment:
I have mixed feelings about this approach. I like the idea but:

* Why do include the division structure inside the `SplitAndTranspose` class instead of inside the `StatCanParser`
* I think that maybe is better to have a tuple (column, prefix) and prefix should be a regexp instead a literal
* Maybe we could make a new class and/or at least one method to check this scenerio

What do you think?